services/state/token/account.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+import "crypto_approve_allowance.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * First-draft representation of a Hedera Token Service account entity in the network Merkle tree.
+ *
+ * As with all network entities, an account has a unique entity number, which is usually given along
+ * with the network's shard and realm in the form of a shard.realm.number id.
+ */
+
+message Account {
+    /**
+     * The unique entity number of the account.
+     */
+    int64 account_number = 1;
+    /**
+     * The public key bytes to be used as the account's alias, if any.
+     */
+    bytes alias = 2;
+    /**
+     * (Optional) The key to be used to sign transactions from the account, if any.
+     */
+    Key key = 3;
+    /**
+     * The expiration time of the account, in seconds since the epoch.
+     */
+    int64 expiry = 4;
+    /**
+     * The balance of the account, in tiny-bars.
+     */
+    int64 hbar_balance = 5;
```

Comment:
There's a few spots we can be more expressive about the units of a quantity...maybe here `tinybar_balance`?

Reply:
Yes, done. Thanks !